### PR TITLE
Mobile branch

### PR DESCRIPTION
**Pull request by Alabi12**
add some CSS  and JavaScript files
work on the linter errors

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -641,3 +742,20 @@ form > ul {
...
+.mobile-menu {
+  background: #6070ff;
+  opacity: 0.888;
+  height: 100%;
+  width: 100%;
+  z-index: 9;
+  position: fixed;
+  display: none;
+}
+
```
Comment (Bondok6) [inline portfolio.css]: - [ ] According to the requirements, The mobile menu should  follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) (e.g., font, colors, images, text, margins), so kindly increase the font-size, the space between the menu-items, and change the color to white to match your Figma design 👍💯

| Your Design  | Figma Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi Team 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Great job implementing the mobile-menu✅
- No Linter errors ✅
- Correct GitHub flow ✅
- Your repo is professional ✅


## Required Changes ♻️
- [ ] _Check the comments under the review._ ⬇

## Optional suggestions
- [OPTIONAL] I noticed that there are two empty files called `a`, and `ul`,  I suggest removing them to have a clean structure 👌
image

- [OPTIONAL] I noticed that you used `onclick` in your html file,  I think it would be better to keep all the JavaScript code in JS files. you can select the element and use `addEventListener` instead 💯

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.css [hunk 2]
```diff
@@ -641,3 +732,26 @@ form > ul {
...
 }
+
+.mobile-menu {
+  background: #6070ff;
+  opacity: 0.888;
+  height: 100%;
+  width: 100%;
+  z-index: 9;
+  position: fixed;
+}
```
Comment (Nemwel-Boniface) [inline portfolio.css:754]: - [ ] As requested by the previous reviewer, kindly ensure that you use the correct margins and padding in your design. This way your design will match the chosen Figma design.

**This is how I am seeing your design right now:**

image

**This is the desired design:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Alabi12     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark:

Hi @Alabi12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-   No linter errors. :heavy_check_mark: 
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark:
-   Milestone worked as expected. :heavy_check_mark:  




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**